### DISCUSSION TITLE
Event show wrong duration.

### DISCUSSION THREAD

**Discussion by phlnil** [blakeblackshear/frigate#12551] [General]
Hi, 
I found an event where Frigate shows a duration of "( 1h 25m 29s )" but if I click on it shows a simple short clip as usually for that cam.
In the log clicking on that event no error is displayed, yet "in progress" another recording... but no this one above.
Thank you.

**Comment by NickM-27** [comment]
It's not the wrong duration. Most likely there is a false positive detected so the event continued longer then the object was there. The recording configuration meant only the true object was recorded. 

This type of issue is handled much better in the next version of Frigate with a redesigned ui

**Reply by phlnil** [reply]
Thank you! NickM-27, I hope to install soon the next release.
In this case I was thinking, indeed,  rather of my misundersting the process of reording.
